blog/2024-08-27-changed-icon-translations-schema.md

```diff
@@ -0,0 +1,33 @@
+---
+author: Erik Montnemery
+authorURL: https://github.com/emontnemery
+title: "Changes to the icon translations schema"
+---
+
+The icon translations schema has been adjusted to allow assigning icons for sections in services.
+Icons for services can now be provided both according to the original schema, which only allows specifying a single icon for the service, and according to a more explicit schema which allows specifying icons for sections.
+
+This allows specifying service icons like this:
+```json
+  "services": {
+    "test_service_1": {
+      "service": "mdi:flask",
+      "sections": {
+        "section_1": "mdi:test-tube"
+      }
+    }
+  }
+```
+
+The old format is still supported:
+```json
+  "services": {
+    "randomize_device_tracker_data": "mdi:dice-multiple"
```

Comment:
Probably better use the same naming for the service name and the icon as in the "new format" example?

Reply:
Yup, fixed 👍